vaxis.go

```diff
@@ -192,6 +194,10 @@ func New(opts Options) (*Vaxis, error) {
 		vx.disableMouse = true
 	}
 
+	if opts.NoSignals {
+		vx.noSignals = true
+	}
```

Comment:
This one doesn't need to be a branch. Can be just `vx.noSignals = opts.NoSignals`